<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Velikost bodu</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spSize"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Počet bodů</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spPoints">
          <property name="maximum">
           <number>200</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Počet centroidů</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spCentroids"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Soubor</string>
    </property>
    <addaction name="actionNacti_body"/>
   </widget>
   <widget class="QMenu" name="menuMo_nosti">
    <property name="title">
     <string>Možnosti</string>
    </property>
    <addaction name="actionGeneruj_body"/>
    <addaction name="actionGeneruj_Centroidy"/>
    <addaction name="separator"/>
    <addaction name="actionVyres"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMo_nosti"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNacti_body"/>
   <addaction name="actionGeneruj_body"/>
   <addaction name="actionGeneruj_Centroidy"/>
   <addaction name="actionVyres"/>
  </widget>
  <action name="actionNacti_body">
   <property name="text">
    <string>Načti body</string>
   </property>
  </action>
  <action name="actionGeneruj_body">
   <property name="text">
    <string>Generuj body</string>
   </property>
  </action>
  <action name="actionVyres">
   <property name="text">
    <string>Vyřeš</string>
   </property>
  </action>
  <action name="actionGeneruj_Centroidy">
   <property name="text">
    <string>Generuj Centroidy</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
